README.md

```diff
@@ -129,11 +129,67 @@ kubetest --test=true \
   --test_args=--node-os-distro=windows --ginkgo.focus=\\[Conformance\\]|\\[NodeConformance\\]|\\[sig-windows\\]|\\[sig-apps\\].CronJob --ginkgo.skip=\\[LinuxOnly\\]|\\[k8s.io\\].Pods.*should.cap.back-off.at.MaxContainerBackOff.\\[Slow\\]\\[NodeConformance\\]|\\[k8s.io\\].Pods.*should.have.their.auto-restart.back-off.timer.reset.on.image.update.\\[Slow\\]\\[NodeConformance\\]
 ```
 
+## Running unit test
+
+Note: This assumes the Windows node is running on GCE, but should be applicable
```

Comment:
@PatrickLang some of content (e.g., setting up `git`, etc) fits better in a development guide (e.g., https://github.com/kubernetes/community/blob/master/contributors/devel/development.md).

I don't think there's one for Windows, so it seems okay to add it here. Let us know if you think otherwise.